Initial

nikpivkin opened the pull request.

nikpivkin committed:
init

afdesk commented on vuln_checker/cli.py:
```diff
@@ -0,0 +1,43 @@
+    models = load_vulnerabilities(args.databases_path, args.databases)
+
+
+if __name__ == "__main__":
```
we don't need this `cli` file, right?

nikpivkin committed:
refactor: remove cli

nikpivkin committed:
refactor: simplify vulnerability model

Make the model flat and reduce the number of fields. Save each model in a json file to a database directory.

nikpivkin committed:
feat: add the redhat database loader

nikpivkin committed:
build: fix the script run command

nikpivkin committed:
fix: alias must be a set

nikpivkin committed:
build: change email
